### PR TITLE
violin87-violinPlaying

### PR DESCRIPTION
**Pull request by violin87**
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did. 
I created animation. Bow playing on violin and background-color is changing

## Pull request checklist

- [V] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [V ] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [V ] Your pull request contains `index.html` and `styles.css`.
	- [V ] Your html file is named exactly: `index.html`
	- [V ] Your css file is named exactly: `styles.css`
- [V ] Your pull request your `meta.json` with:
	- [V] `githubHandle`: being your unique github user name,
	- [V] `artName` : name of your animation
- [V] Your submission includes **at least one animation**.
- [V ] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [ ] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

**Review by cadegore — CHANGES_REQUESTED** [review]
@violin87 Please review the feedback below and update your commits

- [ ] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [x] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [ ] Your pull request contains `index.html` and `styles.css`.
	- [x] Your html file is named exactly: `index.html`
	- [x] Your css file is named exactly: `styles.css`
- [ ] Your pull request your `meta.json` with:
	- [ ] `githubHandle`: being your unique github user name,
	- [ ] `artName` : name of your animation
- [x] Your submission includes **at least one animation**.
- [ ] Your pull request **does not modify any other files** in the repository.

**Comment by LaurelineP** [comment]
Aloha and thanks for the contributions.
Please address the feedback above but also please remove any changes you have outside of your folder

Note:
- your contribution has extra files ( pictures ) we cannot accept
- your contribution needs a `meta.json`

**Comment by violin87** [comment]
I did a new pull request to be qualify for submission (i delete icons and rearrange my program). Please let me know if it works

File: Art/violin87-activeSquares/icon.png
Comment (LaurelineP) [inline Art/violin87-activeSquares/icon.png:1]: ## Picture not accepted 
Please remove this file

File: Art/violin87-activeSquares/styles.css
Comment (LaurelineP) [inline Art/violin87-activeSquares/styles.css:1]: ## CSS format
- [ ] Please remove all empty lines in the file

**Review by LaurelineP — COMMENTED** [review]
Please address the feedbacks and also this one:
you are missing a `meta.json` file 
Please add the file and the values for githubHandle and artName
( see the  [readme code](https://github.com/zero-to-mastery/Animation-Nation)

**Comment by stilt0n** [comment]
It sounds like this was addressed in a new pull request. This branch is also stale so I am going to close it.

If you feel like this should not be closed, feel free to address the code review feedback and re-open it.

**Pull request closed without merging**